contracts/ccip/ccip_onramp/sources/onramp.move

```diff
@@ -893,7 +889,7 @@ fun verify_sender(state: &OnRampState, dest_chain_selector: u64, sender: address
     assert!(state.dest_chain_configs.contains(dest_chain_selector), EUnknownDestChainSelector);
 
     let dest_chain_config = &state.dest_chain_configs[dest_chain_selector];
-    assert!(dest_chain_config.is_enabled, EDestChainNotEnabled);
+    assert!(dest_chain_config.router != @0x0, EDestChainNotEnabled);
```

Comment:
with this update, the router address becomes a signal if this onramp is enabled.
we need to update the deployment script in TS and CLD.